presto-native-execution/README.md

```diff
@@ -96,17 +85,90 @@ checks.  When we need one it can be added.
+  * Working directory: `$MODULE_DIR$`
+  * Environment variables: `PRESTO_SERVER=/Users/spershin/git/presto/presto-native-execution/cmake-build-debug/presto_cpp/main/presto_server;DATA_DIR=/Users/spershin/Desktop/data;WORKER_COUNT=0`
+  * Use classpath of module: choose `presto-native-execution` module.
+* Edit create `TestHiveQueriesUsingJSON` Test Run/Debug Configuration (alter paths accordingly).
+  * Class: `com.facebook.presto.nativeworker.TestHiveQueriesUsingJSON`
+  * VM Options: `-ea -DPRESTO_SERVER=/Users/spershin/git/presto_cpp/cmake-build-debug/presto_cpp/main/presto_server -DDATA_DIR=/Users/spershin/Desktop/data`
+  * Working directory: `$MODULE_WORKING_DIR$`
+
+In "presto/presto-native-execution" run:
+* `make -C presto-native-execution submodules`
+* `git/presto/presto-native-execution % sudo chown -R $(whoami) /usr/local/bin /usr/local/lib /usr/local/sbin; sudo chown -R $(whoami) /usr/local/bin /usr/local/lib /usr/local/bin; sudo chown -R $(whoami) /usr/local/bin /usr/local/lib /usr/local/lib; chmod u+w /usr/local/bin /usr/local/lib /usr/local/sbin; chmod u+w /usr/local/bin /usr/local/lib /usr/local/bin; chmod u+w /usr/local/bin /usr/local/lib /usr/local/lib; ./scripts/setup-macos.sh`
+  
+Run CLion:
+* File->Close Project if any is open.
+* Open `presto/presto-native-execution` folder as CMake project and wait till CLion loads/generates make files, symbols, etc.
+* Edit configuration for `presto_server` module (alter paths accordingly).
+  * Program arguments: `--logtostderr=1 --v=1 --etc_dir=/Users/spershin/git/presto/presto-native-execution/etc`
+  * Working directory: `/Users/spershin/git/presto/presto-native-execution`
+* Edit menu CLion->Preferences->Build, Execution, Deployment->CMake
+  * CMake options: `-DVELOX_BUILD_TESTING=ON -DCMAKE_BUILD_TYPE=Debug`
+  * Build options: `-- -j 12`
+* Edit menu CLion->Preferences->Editor->Code Style->C/C++
+  * Scheme: `Project`
+* To enable clang format you need
+  * Open any h or cpp file in the editor and select `Enable ClangFormat` by clicking `4 spaces` rectangle in the status bar (bottom right) which is next to `UTF-8` bar.
+
+### Using local Presto environment on MacOs (with optional IntelliJ and CLion)
+Running Presto Coordinator + Worker on mac
+* Note that everything below can be done w/o using IDEs by running command line commands (not in this readme).
+* Run 'HiveQueryRunnerExternal' from IntelliJ and wait until it started (`======== SERVER STARTED ========` in the log output).
+* Scroll up the log output and find `Discovery URL http://127.0.0.1:50555`. The port is 'random' with every start.
+* Copy that port (or the whole URL) to the `discovery.uri` field in `presto/presto-native-execution/etc/config.properties` for the worker to Discover the Coordinator.
+* In CLion run "presto_server" module. Connection success will be indicated by `Announcement succeeded: 202` line in the log output.
+* In command line from presto root folder run the presto client:
```

Comment:
I don't run command line stuff from IDE, so I would rather not explain people how to do it. :)

Reply:
Hmm... there might be some misunderstanding. When working in in IDEA it is convenient to avoid switching back and forth between IDEA and command line. Hence, there is a way to run Presto CLI from the IDEA itself. I suggest we document that process here instead of a switch to command line.

<img width="965" alt="Screen Shot 2022-08-30 at 11 17 38 AM" src="https://user-images.githubusercontent.com/27965151/187475448-f231d38b-4da1-4cf3-b864-7519f32d52b8.png">
<img width="1022" alt="Screen Shot 2022-08-30 at 11 17 45 AM" src="https://user-images.githubusercontent.com/27965151/187475452-2a069189-0013-4070-9a29-638aaf34218f.png">

Reply:
It is convenient for some people to run cmd line in IDE.
I prefer switching.
I also expect people chose their way to use cmd line. :)